prjxray/node_lookup.py

```diff
@@ -120,3 +129,4 @@ def wires_for_tile(self, tile):
 """, (tile, ))
         for row in c:
             yield row[0][len(tile) + 1:]
+THIS IS A MODIFICATION
```

Comment:
What is going on here?

Reply:
I don't remember that...It must have been a comment or something.